pkg/agent/openflow/pipeline.go

```diff
@@ -582,6 +695,19 @@ func (c *client) gatewayIPSpoofGuardFlow(gatewayOFPort uint32, category cookie.C
 		Done()
 }
 
+// sessionAffinityReselectFlow generates the flow resubmit the service accessing
+// packet back to serviceLBTable if there is no endpointDNAT flow matched. This
+// case will be occur if an Endpoint is removed and is the learned Endpoint
+// selection of the Service.
+func (c *client) sessionAffinityReselectFlow(category cookie.Category) binding.Flow {
+	return c.pipeline[endpointDNATTable].BuildFlow(priorityLow).
+		MatchRegRange(int(serviceLearnReg), marksRegServiceHitted, serviceLearnRegRange).
+		Action().LoadRegRange(int(serviceLearnReg), marksRegServiceNeedLB, serviceLearnRegRange).
+		Action().ResubmitToTable(serviceLBTable).
```

Comment:
Shall we use "goto_table" instead of "resubmit" here?

Reply:
I'm afraid not here. This flow resubmit packet back to serviceLBTable which has lower table ID.